floating-buttons.html

```diff
@@ -95,8 +95,8 @@
 			},
 
 			ready: function() {
-				this._container = Polymer.dom(this.root).querySelector('.container');
-				this._spacer = Polymer.dom(this.root).querySelector('.spacer');
+				this._container = Polymer.dom(this.root).querySelector('.vui-floating-buttons-container');
+				this._spacer = Polymer.dom(this.root).querySelector('.vui-floating-buttons-spacer');
```

Comment:
I think you can use a shortcut for this: ``this.$$('.vui-floating-buttons-container')``
https://www.polymer-project.org/1.0/docs/devguide/local-dom.html#node-finding